MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -293,6 +293,7 @@
 		B430D2FA25534FDC0088CC23 /* UserHaloCourseView.swift in Sources */ = {isa = PBXBuildFile; fileRef = B430D2F925534FDC0088CC23 /* UserHaloCourseView.swift */; };
 		B48CEFC125796FD300696BB3 /* route-for-vanishing-route-line.json in Resources */ = {isa = PBXBuildFile; fileRef = B48CEFAB25796F5A00696BB3 /* route-for-vanishing-route-line.json */; };
 		B4F2BF3C25FAF69500D7AE2E /* NavigationPlotter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3559FE4E21C3195700B6613F /* NavigationPlotter.swift */; };
+		B4F9FBDF260A61F8008C3D78 /* MapboxCoreNavigation.h in Headers */ = {isa = PBXBuildFile; fileRef = C5ADFBCC1DDCC7840011824B /* MapboxCoreNavigation.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
It’s surprising that MapboxCoreNavigation.h would be added to the TestHelper target. That’s the umbrella header for the MapboxCoreNavigation target and would only be useful for Carthage-based builds. (An umbrella header shouldn’t part of any target other than its own module.)

Reply:
Thanks for the suggestion,  I'll remove this from the `TestHelper` target. Because sometimes the `TestHelper` still could not find the header of the `MapboxCoreNavigation`, and it would cause the error in the local building.